src/prettytable/prettytable.py

```diff
@@ -1836,6 +1839,8 @@ def _stringify_row(self, row, options, hrule):
             for line in lines:
                 if line == "None" and self.none_format.get(field) is not None:
                     line = self.none_format[field]
+                    if len(line) > width:
+                        width = len(line)
```

Comment:
This line isn't covered by tests, is possible to add one?

Reply:
I think the easiest way to test that would be to move it to its own function. In fact, maybe I should just move that entire statement to its own function and test the function individually. Do you think that would be okay?

Reply:
In this case, as it's just a couple of lines, I think it'd be better to test via the public API so we know that this code is actually needed.

What sort of input would trigger this line?

Reply:
I think the check is redundant, and I am pretty sure it can be removed. I did so on my end and all of the tests still pass. The width is recalculated inside of `_compute_widths`.

Reply:
Thanks for checking, let's remove it then!

Reply:
Should now be fixed with 05f0d707.